src/MusicTheory/Duration.elm

```diff
@@ -0,0 +1,37 @@
+module MusicTheory.Duration exposing
+    ( Division(..)
+    , Duration(..)
+    , Grouping(..)
+    , TiedOrSeparate(..)
+    )
+
+
+type Grouping
+    = Single Duration
```

Comment:
In this current version you could put a triple dotted duration into a quintuplet.

Should we allow that?

Reply:
I agree that it would be pretty rare. But I think it's technically valid, right? I can put a triple-dotted sixteenth-note quintuplet in a measure of 4/4 in Sibelius.

Reply:
Yeah, I think it's the responsibility of the user.